waggle-dance-core/src/test/java/com/hotels/bdp/waggledance/metrics/MonitoringConfigurationTest.java

```diff
@@ -1,77 +0,0 @@
-/**
```

Comment:
We still have a MonitoringConfiguration class right? So surely there is some stuff we can test on it?

Reply:
Being handled in another conversation below.